src/Parser/Shaman/Restoration/Modules/Features/AlwaysBeCasting.js

```diff
@@ -25,24 +25,29 @@ class AlwaysBeCasting extends CoreAlwaysBeCastingHealing {
   static HEALING_ABILITIES_ON_GCD = HEALING_ABILITIES_ON_GCD;
   static ABILITIES_ON_GCD = [
     ...HEALING_ABILITIES_ON_GCD,
-    192063, // Gust of Wind
-    192077, // Wind Rush Totem
-    192058, // Lightning Surge Totem
-    51485, // Earthgrab totem
-    196932, // Voodoo totem
-    207399, // APT
-    403, // Lightning Bolt
-    188838, // Flame shock
-    2484, // Earthbind totem
-    51505, // Lava Burst
-    6196, // Far sight
-    2645, // Ghost wolf
-    77130, // Purify spirit
-    421, // Chain lightning
-    546, // water walking
-    211004, 51514, 211010, 210873, 211015, // Variants of hex
-    556, // Astral recall
-    370, // purge
+    SPELLS.GUST_OF_WIND_TALENT.id,
```

Comment:
Setting abilities on GCD through this array is deprecated in favor of the [isOnGcd](https://github.com/WoWAnalyzer/WoWAnalyzer/wiki/Abilities#isongcd--bool-default-false) prop of Abilities.

Reply:
good to know, thought it was odd having to add GCDs in 2 places but just replaced the hardcoded so i don't break stuff